spec/requests/api/v1/targets/destroy_spec.rb

```diff
@@ -0,0 +1,54 @@
+describe 'DELETE api/v1/targets/{id}', type: :request do
+  let!(:user)               { create(:user) }
+  let!(:user_targets)       { create_list(:target, 3, user:) }
+  let!(:other_targets)      { create_list(:target, 3) }
+  let(:first_user_target)   { user_targets.first }
+  let(:failed_response)     { 404 }
+  let(:id)                  { first_user_target.id }
+  subject { delete api_v1_target_path(id:), headers: auth_headers, as: :json }
+
+  it 'returns a successful response' do
+    subject
+    expect(response).to be_successful
+  end
+
+  it 'deletes the target' do
+    expect { subject }.to change(Target, :count).from(6).to(5)
+  end
+
+  it 'returns the deleted target in correct format' do
+    subject
+    expect(json[:target][:id]).to eq(first_user_target.id)
+    expect(json[:target][:title]).to eq(first_user_target.title)
+    expect(json[:target][:radius]).to eq(first_user_target.radius)
+    expect(json[:target][:lat].round(12)).to eq(first_user_target.lat.round(12))
+    expect(json[:target][:lng].round(12)).to eq(first_user_target.lng.round(12))
+    expect(json[:target][:topic_id]).to eq(first_user_target.topic_id)
+  end
+
+  context 'when target do not belongs to current user' do
+    let(:id) { other_targets.first.id }
+
+    it 'does not delete a target' do
+      expect { subject }.not_to change { Target.count }
+    end
+
+    it 'does not return a successful response' do
+      subject
+      expect(response.status).to eq(failed_response)
```

Comment:
```suggestion
      expect(response).to be_not_found
```
Does this work?

Reply:
I tried it before, but I obtained this error:
`Failure/Error: expect(response.status).to be_not_found`
      `expected 404 to respond to `not_found?``

Reply:
Looks like you were doing expect(response.status) instead of expect(response) then 🤔

Reply:
You are right, I don´t see that 🤦‍♂️. Doing `expect(response).to be_not_found` works correctly. It is ok if I change it in the next pr?